### DISCUSSION TITLE
Using lerna to publish the packages of a Nx repo

### DISCUSSION THREAD

**Discussion by matheo** [lerna/lerna#3628] [Q&A]
Hi all!
I'm trying to setup `lerna` to publish the packages in one commit because I don't like the `semver` plugin for Nx
but I'm wondering what's the workflow of the latest `lerna` after bump the version.

Does it bump the `package.json` and sync the build.output manifest to publish it?
I'm trying to see the `verbose` log-level of `lerna publish` and it seems to pack my `src/` code instead `dist/`

Any help is appreciated.

**Comment by fahslaj** [comment]
Hi @matheo , you may need to specify `dist/` as a directory to be included in the publish. You can do this by setting the `files` property in package.json (see the [npm docs](https://docs.npmjs.com/cli/v8/commands/npm-publish#files-included-in-package) for details).

You can actually look at the lerna repo itself for an example of this: https://github.com/lerna/lerna/blob/main/packages/lerna/package.json

**Reply by matheo** [reply]
Thanks for pointing it out @fahslaj 
Given the `@nrwl/angular:package` executor options attached to the `dist/package.json` it seems not proper for my case
I will try to hook again to the `version` lifecycle step and sync the `package.json` file to publish it properly.
